app/backend/src/clj/teet/file/file_commands.clj

```diff
@@ -241,9 +255,11 @@
    :payload {:keys [file-id]}
    :project-id (project-db/file-project-id db file-id)
    :authorization {:document/delete-document {:db/id file-id}}
-   :transact (vec
-              (for [version-id (file-db/file-versions db file-id)]
-                (deletion-tx user version-id)))})
+   :transact (let [file-delete-tx (vec
+                              (for [version-id (file-db/file-versions db file-id)]
+                                (deletion-tx user version-id)))]
+               (maybe-vektorio-delete! db file-id)
```

Comment:
should all versions be deleted? or do later versions update the same model in vektorio?

Reply:
I've understood that we update the same model in vektorio.